contrib/appimage/AppImageBuilder.yml

```diff
@@ -0,0 +1,74 @@
+version: 1
+AppDir:
+  path: AppDir
+  app_info:
+    id: peercoin-qt
+    name: Peercoin-Qt
+    icon: peercoin
+    version: latest
+    exec: bin/peercoin-qt
+    exec_args: $@
+  runtime:
+    env:
+      APPDIR_LIBRARY_PATH: $APPDIR/lib
+  apt:
+    arch: !ENV ${APT_ARCH}
+    sources:
+      - sourceline: 'deb http://archive.ubuntu.com/ubuntu/ bionic main restricted universe multiverse'
+        key_url: 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3b4fe6acc0b21f32'
+      - sourceline: 'deb http://archive.ubuntu.com/ubuntu/ bionic-updates main restricted universe multiverse'
+      - sourceline: 'deb http://archive.ubuntu.com/ubuntu/ bionic-backports main restricted universe multiverse'
+    include:
+      - libc6
+      - libzmq5
+      # Full bundle requirements
+      - libx11-6
+      - libgl1
+      - libglapi-mesa
+      - libdrm2
+      - libegl1
+      - libxcb-shape0
+      - libxcb1
+      - libx11-xcb1
+      - fontconfig-config
+      - libfontconfig1
+      - libfreetype6
+      - libglx0
+      - libxcb-xfixes0
+      - libxcb-render0
+      - libxcb-glx0
+      - libxcb-shm0
+      - libglvnd0
+      - libxcb-dri3-0
+      - libxcb-dri2-0
+      - libxcb-present0
+    exclude:
+      - phonon4qt5
+      - libkf5service-bin
+      - libpam-runtime
+      - hicolor-icon-theme
+      - sound-theme-freedesktop
+      - perl
+      - perl-base
+      - kwayland-data
+      - libwacom2
+      - libasound2
+      - breeze
+      - breeze-icon-theme
+      - breeze-cursor-theme
+      - kwin-style-breeze
+      - kde-style-breeze
+      - plasma-integration
+  files:
+    exclude:
+      - usr/share/man
+      - usr/share/doc/*/README.*
+      - usr/share/doc/*/changelog.*
+      - usr/share/doc/*/NEWS.*
+      - usr/share/doc/*/TODO.*
+AppImage:
+  arch: !ENV ${BUILD_ARCH}
+  # export UPDATE_INFO="zsync|https://gitlab..."
+  # !ENV ${UPDATE_INFO}
```

Comment:
I left out on purpose. This needs to be added by whoever has the say over it.

Reply:
what exactly should these lines do. Hard to tell who has a say over it without that info :)

Reply:
From what I understand, `update-information` gives a location to a zsync meta file, which carries, well, update information about the package.  
It is being used here https://github.com/peercoin/peercoin/blob/e4f744c/.github/workflows/appimage.yml#L23 already, but might need to be changed to wherever you end up hosting the AppImage files.

Reply:
git blame points to @peerchemist this commit: 06c8861fdea63432e800071d4ee6342e16f5c3da 
Should that this info should be synced to the one from appimage.yml, or vice versa?

Reply:
We can undo that commit, that was basically my experiment.

Reply:
Removed that file.